clearwater-infrastructure/usr/share/clearwater/bin/view-subscriber

```diff
@@ -69,8 +69,12 @@ def main():
 
                 ims_subscription = ET.tostring(reg_data.find('IMSSubscription'))
 
-                ccf_addresses = [(ccf.text, ccf.attrib['priority']) for ccf in reg_data.find('ChargingAddresses').findall('CCF')]
-                ecf_addresses = [(ecf.text, ecf.attrib['priority']) for ecf in reg_data.find('ChargingAddresses').findall('ECF')]
+                if reg_data.find('ChargingAddresses'):
```

Comment:
This isn't a spec-mandated field, this is something Homestead adds to get all the relevant info from the SAR into the XML, and it only does so if there are charging addresses - see https://github.com/Metaswitch/homestead/blob/e16d635eee87899cfefbc8f6fa3c35ad8b170215/src/xmlutils.cpp#L125.